lib/features/uome2/presentation/bloc/auth/auth_event.dart

```diff
@@ -0,0 +1,8 @@
+part of 'auth_bloc.dart';
```

Comment:
We can ditch events and use cubits at some point since events are mostly boilerplate. We can discuss this somewhere down the road.